### PR TITLE
Create ProLiant_DL360_Gen10.yaml

### PR DESCRIPTION
**Pull request by bbrk364**
Adding a new profile for HPE DL360 Gen

### CODE REVIEW

File: device-types/HPE/ProLiant_DL360_Gen10.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - name: Rear Optional Serial
+    type: de-9
+  - name: Rear VGA
+    type: other
+  - name: Front USB1
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL360_Gen10.yaml:21]: We are no longer modelling USB ports as console ports.  Please remove

File: device-types/HPE/ProLiant_DL360_Gen10.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+manufacturer: HPE
+model: ProLiant DL360 Gen10 8SFF 
+slug: hpe_dl360_g10
+part_number: P19766-B21
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: Front iLO Front Service Port
+    type: usb-a
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL360_Gen10.yaml:11]: Should be modelled as a IP interface as that is how this functions.

File: device-types/HPE/ProLiant_DL360_Gen10.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    type: 1000base-t
+    mgmt_only: false
+  - name: iLO
+    type: 1000base-t
+    mgmt_only: true
+device-bays:
+  - name: FlexibleLOM2 
+  - name: PCIe 1
+  - name: PCIe 2
+  - name: Option PCIe 3
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL360_Gen10.yaml:49]: Should be modelled as a module bay

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @bbrk364

**Pull request closed without merging**